dev/com.ibm.ws.kernel.service.location/src/com/ibm/ws/kernel/service/location/internal/WsLocationAdminImpl.java

```diff
@@ -66,7 +68,7 @@
                     LOC_AREA_NAME_WORKING = "workarea/",
                     LOC_AREA_NAME_EXTENSION = "extension/";
 
-    private static WsLocationAdminImpl instance;
+    private volatile static StaticValue<WsLocationAdminImpl> instance = StaticValue.createStaticValue(null);
```

Comment:
Certainly not necessary to do in this work item, but it would be nice to get rid of some of these statics altogether.

Reply:
Yes, would like to get rid of them all, but will take a long time to unwind all the procedural patterns which are using the statics.